### ISSUE TITLE
Determine if an image falls under open access policy 

### ISSUE THREAD

**Issue by Sgambe33** [NationalGalleryOfArt/opendata#10]
Hi, is it possible in any way to automatically determine if an image in pusblished_images.csv falls under open access policy so that it can be used freely?

**Comment by w8HAQRHkTx7r** [comment]
Is this issue related to "Imagery for an object should not be displayed unless the canShowImagery flag is 1"?  The statement appears in the data dictionary text, but not as a field in any table that I can find.   TIA

**Comment by beaudet** [comment]
Yes, it is very much related but "canshowimagery" refers to an internal deprecated flag.  It should not be in the published version of the data dictionary.  We recently discussed this github issue and will be releasing new flags in the published images table to indicate which images fall under our open data program and therefore a permissive content license.

**Comment by SamiNorling** [comment]
@beaudet This question came up in the IIIF slack yesterday as to whether there is a way in the repo to determine which images are open for use. Any update on adding the flag to the published images table, or is there somewhere else people should look for that?
